wirepas_mesh_messaging/status.py

```diff
@@ -42,12 +50,28 @@ class StatusEvent(Event):
         state (GatewayState): state of the gateway
         version (int): protobuf messsage definition version for the gateway. This implementation can only parse protobuf message definitions in version1
         event_id(int): event unique id (random value generated if None)
+        sink_configs (list): list of dictionnary containing the sink configs
+        gateway_model (string): gateway model
+        gateway_version (string): gateway version
     """
 
-    def __init__(self, gw_id, state, version=PB_MESSAGE_DEFINITION_VERSION, event_id=None, **kwargs):
+    def __init__(
+        self,
+        gw_id,
+        state,
+        version=PB_MESSAGE_DEFINITION_VERSION,
+        event_id=None,
+        sink_configs=[],
```

Comment:
It might be very dangerous to put a mutable as a default value of a method. Two instances of this class can have the same list address for this attribute which is not desirable.

Reply:
Hum, yes you are right. Never mind, we are not supposed to use this default value nor changing it afterwards, my point is not relevant.